src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -515,5 +517,29 @@ public override void Shutdown()
             Stop();
             HostingEnvironment.InitiateShutdown();
         }
+
+        public async Task DelayUntilHostReady(int timeoutSeconds = WebScriptHostHandler.HostTimeoutSeconds, int pollingIntervalMilliseconds = WebScriptHostHandler.HostPollingIntervalMilliseconds, bool throwOnFailure = true)
```

Comment:
Looks pretty straightforward -- this will just let Web API and IIS do their natural queueing when requests take longer to process, right?